data.py

```diff
@@ -0,0 +1,74 @@
+def load_data(file_path):
```

Comment:
this function is very long
make sure you modularize it
use classes, split into helper functions